Changelog.md

```diff
@@ -1,10 +1,15 @@
-[0.11.0.0] —
- * [Add `indexMaybe` and synonym `(!?)` for indexing that returns `Maybe`](https://github.com/haskell/bytestring/pull/261)
- * Add `unsafePackLiteral` to `Data.ByteString.Internal`. When possible, use
+[0.11.0.0] — September 2020
+ * [Change internal representation of `ByteString`, removing offset](https://github.com/haskell/bytestring/pull/175)
+ * [Remove deprecated functions `findSubstring` and `findSubstrings`](https://github.com/haskell/bytestring/pull/181)
+ * [Speed up sorting of short strings](https://github.com/haskell/bytestring/pull/267)
+ * [Improve handling of literal strings](https://github.com/haskell/bytestring/pull/132)
+ * [Add `unsafePackLiteral` to `Data.ByteString.Internal`](https://github.com/haskell/bytestring/pull/191)
+   When possible, use
    [known-key](https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/compiler/wired-in)
    variant of C's `strlen` from `GHC.CString` that supports constant folding.
    This results in better generated code when an ASCII-only string literal
-   is desugared to `ByteString` by the `OverloadedStrings` extension. ([#191])
+   is desugared to `ByteString` by the `OverloadedStrings` extension.
```

Comment:
Maybe move this description into a (sub)list item or just add a dot after

    Add `unsafePackLiteral` to `Data.ByteString.Internal`

?

Reply:
I wonder whether this entry is too long and too specific for a changelog. My understanding is that the main benefit is not to provide a new function (not that we really expect end users to use it directly). What about 
"Improve opportunities for optimizations when a function scrutinizes the length of a `ByteString`"?

Reply:
Yeah, this could be clearer and probably shorter.

How about

> Compute length of literal strings at compile time
> * This improves optimization opportunities for functions that scrutinize the length of a `ByteString`.

(CC @andrewthad)